core/src/main/resources/hudson/model/StringParameterDefinition/config.jelly

```diff
@@ -35,7 +35,7 @@ THE SOFTWARE.
     <f:entry title="${%Description}" help="/help/parameter/description.html">
         <f:textarea name="parameter.description" value="${instance.description}" codemirror-mode="${app.markupFormatter.codeMirrorMode}" codemirror-config="${app.markupFormatter.codeMirrorConfig}" previewEndpoint="/markupFormatter/previewDescription" />
     </f:entry>
-    <f:entry field="trim" help="/help/parameter/trim.html">
-        <f:checkbox default="false" title="${%Trim the string}"/>
+    <f:entry>
+        <f:checkbox field="trim" help="/help/parameter/trim.html" default="false" title="${%Trim the string}" />
```

Comment:
In https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/62#discussion_r273305172, @jglick requested that the `field` remain on the `entry`, although I'm not entirely sure why. Have you tested to make sure that this particular change works fine with `field` on `checkbox` rather then `entry`?

Reply:
`field` should be on entry. In this case it may not matter now, so long as `help` is being specified explicitly (rather than simply being `help-trim.html` in the same directory), but it is better style.

Reply:
So, `/help/parameter/*` files are used by at least `copyartifact-plugin` and `credentials-plugin`.

The trim file appears to only be used here, so I'm willing to move it:
https://github.com/search?q=org%3Ajenkinsci+%22help%2Fparameter%2Ftrim.html%22&type=Code

-- I'll look into cleaning this up... although it's a bit of an extended mess.

Reply:
Thanks @dwnusbaum / @jglick